drivers/source/serial_io.c

```diff
@@ -19,7 +19,7 @@
 
```

Comment:
Looks like you have some build errors to fix. Btw, you can build your program locally by following the instructions [here](https://www.notion.so/uworbital/Firmware-Onboarding-Challenge-b208ef6d62cd436a8e738dc99f6d2c4d?pvs=4#4cd6898892814e87adbad8122e3004dd).